DodgerGame/src/main/java/Game.java

```diff
@@ -1,3 +1,105 @@
+import Entities.Player.Spaceship;
+import Entities.Player.SpaceshipFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
 public class Game {
+    private static Game instance = null;
+    private Spaceship spaceship = SpaceshipFactory.createSpaceship(true, 368, 268);
+    private Spaceship wrapAroundSpaceship = SpaceshipFactory.createSpaceship(false, 10000, 10000);
+    private List<Spaceship> spaceships = new ArrayList<>();
+
+    private Game() {
+    }
+
+    //@Author Tobias Engblom
+    protected static Game getInstance() {
+        if (instance == null) {
+            instance = new Game();
+            instance.initSpaceships();
```

Comment:
Think you should move initSpaceships to private Game() instead.

Reply:
Okay, will do that!